packages/core/src/components/hotkeys/hotkeys.tsx

```diff
@@ -24,7 +24,7 @@ import { H4 } from "../html/html";
 
 import { Hotkey, type HotkeyProps } from "./hotkey";
 
-export interface HotkeysProps extends Props {
+export interface HotkeysProps extends Props, React.PropsWithChildren<object> {
```

Comment:
It looks like we're expecting children to be `React.ReactElement<HotkeyProps>`. Should we use that type over `object`?

https://github.com/palantir/blueprint/blob/0c815e909e86f511d9b057e7fa4f2dd80c91a3c8/packages/core/src/components/hotkeys/hotkeys.tsx#L57

Or was the goal to preserve the same type definition of `HotkeyProps` and avoid compile breaks on consumers?

Reply:
This one is a nitpick, but I've personally been avoiding `React.PropsWithChildren<...>` since I see the React docs usually write the `children` prop directly.

https://react.dev/learn/typescript#typing-children

Having the `children` prop directly in the interface also makes it easier to add a JSDoc on the field.

Reply:
fd385b9a1e5e30308df5d7637f2f0aa279ae493b